demo/package.json

```diff
@@ -4,11 +4,13 @@
   "version": "0.1.0",
   "dependencies": {
     "@primer/css": "^21.3.6",
-    "@primer/primitives": "^9.1.2",
     "@rails/actioncable": "^7.2.100",
     "@rails/ujs": "^7.1.400",
     "turbolinks": "^5.2.0"
   },
+  "peerDependencies": {
+    "@primer/primitives": "9.x || 10.x"
+  },
```

Comment:
This change in the demo isn't necessary since we don't publish this package, it only effects what's being used in the lookbook

Reply:
Fixed this